### ISSUE TITLE
Log warning about legacy database files in the DB directory

### ISSUE THREAD

**Issue by dhs-rec** [sonatype/nexus-public#476]
After migrating from OrientDB to H2 I see the following warning in the log after every Nexus start:
```
[FelixStartLevel]  *SYSTEM org.sonatype.nexus.datastore.mybatis.MyBatisDataStore - Database directory contains unsupported legacy database files; remove legacy files as soon as possible.
```
The migration documentation doesn't mention any cleanup procedure, so which "legacy files" should be removed "as soon as possible"?

**Comment by dhs-rec** [comment]
I guess everything except `nexus.mv.db` can be removed from the `.../db` directory. Please confirm.

**Comment by paulc0te** [comment]
I have the same problem. Please confirm.

**Comment by bharathjakkani** [comment]
I have recently upgraded the Nexus Sonatype repo from 3.70 to 3.74 after migrating the Orient DB to H2 and saw the same warning on the docker logs of Nexus container. Please confirm, if we can remove all the files except the nexus.mv.db

**Comment by bharathjakkani** [comment]
I believe except the nexus.mv.db all the files should be removed because a temporary container with :latest tag I spinned up shows only that file in the db/ folder. Also, the old container does not shows this warning anymore.

**Comment by dhs-rec** [comment]
@bharathjakkani, yes, that's my impression, too. But I'd like to have an official confirmation from a Nexus developer before I delete anything, esp. so as this is not documented anywhere (at least I couldn't find anything).

**Comment by PhilippHomann** [comment]
Log is produced [here](https://github.com/sonatype/nexus-public/blob/fc7a25764171b2550d7c87a9d4e97d5eaae6a263/components/nexus-datastore-mybatis/src/main/java/org/sonatype/nexus/datastore/mybatis/MyBatisDataStore.java#L424) and is logged because [you have old OrientDB files lying around in db folder](https://github.com/sonatype/nexus-public/blob/fc7a25764171b2550d7c87a9d4e97d5eaae6a263/components/nexus-datastore-mybatis/src/main/java/org/sonatype/nexus/datastore/mybatis/MyBatisDataStore.java#L442).
